app/controllers/admin/application_controller.rb

```diff
@@ -8,7 +8,8 @@ class Admin::ApplicationController < Administrate::ApplicationController
   before_filter :authenticate_admin
 
   def authenticate_admin
-    # TODO Add authentication logic here.
+    return no_access_guest if current_user.nil?
```

Comment:
Since the line below will be using `unless` -- I tend to make multiple guard clauses symmetrical - so you could change this to:

```
return no_access_guest unless current_user.present?
```